libtransact/src/protocol/batch.rs

```diff
@@ -11,7 +11,7 @@ use crate::signing;
 
 use super::transaction::Transaction;
 
-#[derive(Clone)]
+#[derive(Clone, Debug)]
```

Comment:
Why do these need to be made Debug here?

Reply:
This should've been in the next commit; updated